contentctl/objects/correlation_search.py

```diff
@@ -18,10 +19,22 @@
     TestReportingType,
     TestingStates
 )
+from contentctl.objects.errors import (
+    IntegrationTestingError,
+    ServerError,
+    ClientError,
+    ValidationFailed
+)
+from contentctl.objects.detection import Detection
+from contentctl.objects.risk_event import RiskEvent
+from contentctl.objects.notable_event import NotableEvent
+from contentctl.objects.observable import Observable
+
+# TODO: test my new contentctl logic against an old ESCU build; my logic should detect the faulty attacker events
 
 
 # Suppress logging by default; enable for local testing
-ENABLE_LOGGING = False
+ENABLE_LOGGING = True
```

Comment:
Will set this back to False after a final full test w/in an SCA pipeline

Reply:
@cmcginley-splunk : Should we set this back to false now that we have ran this a few times in SCA repo?